### DISCUSSION TITLE
gh-pages failed build?

### DISCUSSION THREAD

**Discussion by odds-get-evened** [mmistakes/jekyll-theme-basically-basic#150] [Q&A]
I am getting gh-pages build fail when using this theme. Full error from Github:

```
The page build failed for the `gh-pages` branch with the following error:

Your SCSS file `assets/stylesheets/main.scss` has an error on line 11: File to import not found or unreadable: basically-basic/vendor/breakpoint/breakpoint. Load path: _sass. For more information, 
```

I looked at the sass file but it looks okay to me. Any ideas?

**Comment by mmistakes** [comment]
The error is your first clue:

```
Your SCSS file `assets/stylesheets/main.scss` has an error on line 11: File to import not found or unreadable: basically-basic/vendor/breakpoint/breakpoint. Load path: _sass. For more information,
```

All of the Sass partials in[`_sass/basically-basic/vendor/`](https://github.com/mmistakes/jekyll-theme-basically-basic/tree/master/_sass/basically-basic/vendor) are missing in your repo. If I had to guess you have `vendor` gitignored in `.gitignore` and it's not pushing up those files to your GitHub repo. So GH Pages fails when it tries to build since there's a bunch of missing files.
